modules/javafx.controls/src/main/java/javafx/scene/control/skin/TabPaneSkin.java

```diff
@@ -700,7 +713,7 @@ private Timeline createTimeline(final TabHeaderSkin tabRegion, final Duration du
     }
 
     private boolean isHorizontal() {
-        Side tabPosition = getSkinnable().getSide();
+        Side tabPosition = getSkinnable() != null ? getSkinnable().getSide() : Side.TOP;
```

Comment:
I agree with @kleopatra This null check suggests that someone (a listener perhaps?) is calling into the skin after it has been disposed.

Reply:
The NPE here occurs if `tabHeaderArea` is not removed when disposing the skin.
Actually this NPE does not occur with commit#1 and the null check was not needed, It is removed now.
(The NPE can be observed by removing line#283 from TabPaneSkin.java of this fix)